webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartling.java

```diff
@@ -270,52 +273,145 @@ private void processPullBatch(Long batchNumber,
         }
     }
 
+    @Override
+    public void pushTranslations(Repository repository,
+                                 String projectId,
+                                 String pluralSeparator,
+                                 Map<String, String> localeMapping,
+                                 String skipTextUnitsWithPattern,
+                                 String skipAssetsWithPathPattern,
+                                 List<String> optionList) {
+
+        SmartlingOptions options = SmartlingOptions.parseList(optionList);
+        AndroidStringDocumentMapper mapper = new AndroidStringDocumentMapper(pluralSeparator, null);
+
+        repository.getRepositoryLocales()
+                  .stream()
+                  .filter(RepositoryLocale::isToBeFullyTranslated)
```

Comment:
was this in the original script? If so i'm not sure why that's the case: importing locales that are not meant to be fully translated still makes sense (that flag is mostly used to say that's ok to have some string not translated for that locale but it doesn't mean the translation are not good)

Reply:
Updated.